canonical/snapcraft.io
# Remove legacy listing page

## Done
Removed the legacy listing page

## QA
- Go to https://snapcraft-io-4090.demos.haus/SNAP_NAME/listing
- Check that the page works as expected
